babel/src/test/resources/sql/spark.iq

```diff
@@ -0,0 +1,243 @@
+# spark.iq - Babel test for Spark dialect of SQL
```

Comment:
Are you sure this test only takes effect for semantics that conform to spark sql?

Reply:
This test will load Built-in functions and spark functions.

Reply:
I am not against using this test.

But I want to know the necessity of using QuidemTest for Spark.
Because your code may also become a future contribution specification.

Let's discuss two questions.
1. Why need `spark.iq`?
2. What features need to be tested and what features don't need to be tested?

Reply:
+1
At present, calcite has adapted many spark functions. Do we need to add all spark functions? What are the benefits after adding them?

Reply:
1. Compared to `SqlOperatorTest`, I think the function tests in Babel are more accurate and complete. Therefore, 
references`big-query.iq` and `redshift.iq`,  `spark.iq` were added to verify the correctness of function execution.
2. I think that unit tests can be added to this file when adding spark functions, but it is unnecessary.